llvm/test/Jeandle/RISCV/calling-conv.ll

```diff
@@ -0,0 +1,74 @@
+; RUN: llc -O0 -mtriple=riscv64-unknown-linux-gnu < %s | FileCheck %s
+
+define hotspotcc i32 @test_i32_args(i32 %a1, i32 %a2, i32 %a3, i32 %a4, i32 %a5, i32 %a6, i32 %a7, i32 %a8) {
+; CHECK-LABEL: test_i32_args:
+; CHECK:            sd a0, 8(sp)
+; CHECK:            mv a0, a2
+; CHECK:            ld a2, 8(sp)
```

Comment:
These `sd` and `ld`  instructions are strange, but the register allocation seems work as expected.